src/Sulu/Component/Webspace/Loader/schema/webspace/webspace-1.0.xsd

```diff
@@ -130,6 +130,7 @@
         <xs:simpleContent>
             <xs:extension base="xs:string">
                 <xs:attribute name="type" type="xs:string" use="required"/>
+                <xs:attribute name="parent-template" type="xs:string" use="optional"/>
```

Comment:
I think in general we could agree to only add this new feature to `webspace-1.0.xsd` and `XmlFileLoader11.php`.

Reply:
So I should remove this from webspace-1.1.xsd?

Reply:
Sorry, wrote the wrong comment :see_no_evil: We only add this feature to `webspace-1.1.xsd` and `XmlFileLoader11.php`, so it should be removed on the older `webspace-1.0.xsd` and `XmlFileLoader10.php` files.

Reply:
Sorry, mentioned the wrong file, we should only do this on `webspace-1.1.xsd`, and not in this one.

Reply:
Ok, parent template functionality removed from 1.0, added in 1.1